src/main.c

```diff
@@ -189,6 +189,7 @@ int main(int argc, char *argv[]) {
     log_add_stderr(LOG_DEBUG, true);
     log_set_level(LOG_DEBUG);
 #else
+    log_add_stdout(LOG_INFO, true);
```

Comment:
stdout is for data output, stderr is for logging messages.

https://en.wikipedia.org/wiki/Standard_streams

Reply:
Okay, I didn't realize that release build already logs to the log file.